starter/slack-spring-boot/src/main/kotlin/io/olaph/slack/broker/security/VerificationMethodArgumentResolver.kt

```diff
@@ -73,4 +76,17 @@ abstract class VerificationMethodArgumentResolver(private val signingSecret: Str
         }
     }
 
+    /**
+     * since spring decodes unreserved characters automatically
+     * we need to replace them with their url escaped value again
+     * http://www.rfc-editor.org/rfc/rfc1738.txt
```

Comment:
Can you mention that this might not work if one the parameter has an asteriks in it.

Reply:
could you elaborate on that? I am not sure what you mean by that